resources/assets/ts/vedst-scripts.ts

```diff
@@ -475,17 +476,19 @@ $(document).ready(function() {
                 }
 
                 // Fill with data received
-                for (var i = 0; i < response.length; i++)
+                for (let i = 0; i < response.length; i++)
                 {
+                    console.dir(response[i]);
                     $("#person-shifts-overview").append(
                         "<tbody>"
                         // Change background for shifts in other sections
-                        + "<tr" + (!response[i]["isOwnClub"] ? " class=\"active text-muted\"" : "") + ">"
```

Comment:
Replaced the "active" class otherwise the palette background color is not shown.